ui/app/AppLayouts/Chat/controls/community/PermissionItem.qml

```diff
@@ -0,0 +1,248 @@
+import QtQuick 2.3
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Utils 0.1
+
+Control{
+    id: root
+
+    implicitHeight: contentItem.implicitHeight
+    implicitWidth: 560 // by design
```

Comment:
You should set this in the contentItem

Reply:
I will not fix the width in the component, so it is more flexible, and I will leave it to the consumer